### DISCUSSION TITLE
I can not use the extension

### DISCUSSION THREAD

**Discussion by joseluisgs** [vitest-dev/vscode#53] [Q&A] [answered]
Hi
I have this project
https://github.com/joseluisgs/Prueba-Tecnica-Cola-Prioridad-Sin-Repetidos-Kotlin/tree/main/typescript

I have installed the extension and configure it (show image):

But I cant use it or I don't know how

Could you help me?

Thank you very much
![Screenshot_20220710_125820](https://user-images.githubusercontent.com/47913953/178142113-5f82f10d-a4d9-46ab-9d73-b0ffa821156a.png)
![Screenshot_20220710_130009](https://user-images.githubusercontent.com/47913953/178142114-234560f5-80a2-415f-a193-989517fb16c2.png)
![Screenshot_20220710_130029](https://user-images.githubusercontent.com/47913953/178142115-7789e4e5-8ad9-4b45-bb07-5973c2378c88.png)

**Comment by ktdotgit** [comment]
I'm in the same boat. I don't even have anything show up on the side panel.

I'm using vscode w/ a remote SSH if that helps.

**Reply by joseluisgs** [reply]
Any ideas?

**Comment by zxch3n** [comment, marked as answer]
If you open `./typescript` directory in vscode, does the extension work?

**Reply by joseluisgs** [reply]
Hi. where is this directory?

**Reply by zxch3n** [reply]
I got it. The extension currently only tries to activate itself if the project contains `vitest.config.*s` or `vite.config.*s`. But rather, it should try to activate if there is a `package.json`. I already published a fix in the pre-release v0.2.23 version.

**Reply by joseluisgs** [reply]
Great!!! Please, confirm where the release will be ready!!
I would like to try it in simple TS node projects :)

**Reply by zxch3n** [reply]
You can switch to the pre-release version inside the VS Code's Vitest extension page. You can give it a try; if there are issues, please report them so that I can fix them before the release 😊

**Reply by joseluisgs** [reply]
Thank you very much!!
